src/pages/Home/Home.module.css

```
@@ -0,0 +1,135 @@
+  color: #0a0b0d;
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.4), inset 0 0 0 1px rgba(255, 255, 255, 0.1);
+  position: relative;
+}
+.medal[data-tier='1'] { --_medal-top: #ffd58a; --_medal-bottom: #d4af37; }
+.medal[data-tier='2'] { --_medal-top: #ffd3c2; --_medal-bottom: #f9bfa7; }
+.medal[data-tier='3'] { --_medal-top: #f6f7f7; --_medal-bottom: #d6d9da; }
+.medal[data-tier='4'] { --_medal-top: #ffeeae; --_medal-bottom: #fae57e; }
+.medal[data-tier=''] {
+  --_medal-top: #4a5058;
+  --_medal-bottom: #2a2f36;
+  color: var(--fg-1);
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)